static/js/anbox-stream-sdk.js

```diff
@@ -128,25 +128,24 @@ class AnboxStream {
             body: JSON.stringify(details),
         })
         .then(response => {
-            if (response.status !== 200) {
-                this._options.callbacks.error(
-                    new Error("Failed to join instance: " + response.error));
-                return
-            }
+            if (response.status !== 200)
+                throw new Error("Failed to join instance");
```

Comment:
missing `response.error` message

Reply:
Nope, response isn't the json data at this point, it is the raw response object.